modules/boost.scope_exit/1.83.0/MODULE.bazel

```
@@ -0,0 +1,15 @@
+module(
+    name = "boost.scope_exit",
+    version = "1.83.0",
+    bazel_compatibility = [">=7.2.1"],
+    compatibility_level = 108300,
+)
+
+bazel_dep(name = "boost.config", version = "1.83.0")
+bazel_dep(name = "boost.detail", version = "1.83.0")
+bazel_dep(name = "boost.function", version = "1.83.0")
+bazel_dep(name = "boost.mpl", version = "1.83.0")
+bazel_dep(name = "boost.preprocessor", version = "1.83.0")
+bazel_dep(name = "boost.type_traits", version = "1.83.0")
+bazel_dep(name = "boost.typeof", version = "1.83.0")
+bazel_dep(name = "boost.utility", version = "1.83.0")
```

```suggestion
bazel_dep(name = "rules_cc", version = "0.0.13")
```
missing rules_cc dependency